example/simple_testing/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,52 @@
+plugins {
```

Comment:
issue: these kotlin files should be removed, yes?

Reply:
yes, yeeted with the rest of `simple_example` !